packages/svelte/messages/client-warnings/warnings.md

```diff
@@ -1,3 +1,7 @@
+## bad_raw_snippet_render
```

Comment:
we use 'invalid' in comparable messages elsewhere, this would be the only use of `bad_`

```suggestion
## invalid_raw_snippet_render
```